src/App.tsx

```diff
@@ -1,9 +1,19 @@
-import React from 'react';
+import React, { useState } from 'react';
+import moviesFromServer from './api/movies.json';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
-import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const getvisibleMovies = () => {
+    return moviesFromServer.filter(movie => movie.title.toLowerCase()
+      .includes(query.toLowerCase()) || movie.description.toLowerCase()
+      .includes(query.toLowerCase()));
+  };
+
+  const visibleMovies = getvisibleMovies();
```

Comment:
You don't need a separate function here.
```suggestion
  const visibleMovies = moviesFromServer.filter(movie => 
    movie.title.toLowerCase() .includes(query.toLowerCase()) || 
    movie.description.toLowerCase().includes(query.toLowerCase()));
  };
```